src/test/java/com/coherent/testng/JsonParserTNG.java

```diff
@@ -0,0 +1,91 @@
+package com.coherent.testng;
+
+import com.coherent.junit.MyParser;
+import org.junit.jupiter.api.Assertions;
+import org.testng.annotations.*;
+import parser.JsonParser;
+import parser.NoSuchFileException;
+import shop.Cart;
+import shop.RealItem;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class JsonParserTNG {
+
+    Cart allaCart = new Cart("alla-cart-testNG");
```

Comment:
Many times you are using "alla-cart-testNG". Please create class variable and reuse it in methods

Reply:
Resolved